postgres/src/main/java/org/jdbi/v3/postgres/PGObjectColumnMapper.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.postgres;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import org.jdbi.v3.core.mapper.ColumnMapper;
+import org.jdbi.v3.core.statement.StatementContext;
+import org.postgresql.util.PGobject;
+
+/**
+ * Column mapper for {@link PGobject}.
+ */
+public class PGObjectColumnMapper implements ColumnMapper<PGobject> {
```

Comment:
This should probably be a ColumnMapperFactory so that we can explicitly column mappers for `PGobject` _and_ all its registered subtypes.

This way we can also _not_ provide a column mapper for `PGobject` subtypes which have _not_ been registered through `PostgresTypes`.

Also, please make class private